superset/tasks/schedules.py

```diff
@@ -43,6 +43,7 @@
 # Time in seconds, we will wait for the page to load and render
 PAGE_RENDER_WAIT = 30
 
+DATETIME_FORMAT = '%Y/%m/%d/%H/%M/%S'
```

Comment:
nit: This should be in the `config.py` and allow users to set this accordingly

Reply:
@hughhhh thank you for the comment. This is not meant for user configuration at all. This is just to define an interface for passing the task's run time as an argument to a celery task.